examples/docker-compose/notebook/README.md

```diff
@@ -0,0 +1,96 @@
+Run a docker-stack notebook container using Docker Compose on a Docker Machine-controlled host.
+
+## Pre-requisites
+
+* [Docker Engine](https://docs.docker.com/engine/) 1.10.0+
+* [Docker Machine](https://docs.docker.com/machine/) 0.6.0+
+* [Docker Compose](https://docs.docker.com/compose/) 1.6.0+
+
+See the [installation instructions](https://docs.docker.com/engine/installation/) for your environment.
+
+## Quickstart
+
+Here's how to build and run a `jupyter/minimal-notebook` container on an existing Docker machine.
```

Comment:
Can it say "Quickstart" instead of what it says (provisioning docker machines and running notebook servers). I didn't read it as a quickstart at all the first time through.